src/components/wallets/WalletAccordion.tsx

```diff
@@ -0,0 +1,357 @@
+import { useState } from 'react'
+import { Accordion, AccordionSummary, AccordionDetails } from '@mui/material'
+import styled, { css } from 'styled-components'
+import { DateTime } from 'luxon'
+import { gql } from '@apollo/client'
+
+import { theme } from '~/styles'
+import {
+  Avatar,
+  Button,
+  Icon,
+  Skeleton,
+  Status,
+  StatusEnum,
+  Tooltip,
+  Typography,
+} from '~/components/designSystem'
+import { useInternationalization } from '~/hooks/core/useInternationalization'
+import { WalletDetailsFragment, WalletStatusEnum } from '~/generated/graphql'
+
+gql`
+  fragment WalletDetails on Wallet {
```

Comment:
Ok sorry, when possible try to use the same name as the component so we can find where it's defined easily (and for more consistency as well)

Reply:
Done